fetch.bs

```diff
@@ -3241,7 +3253,111 @@ through TLS using ALPN. The protocol cannot be spoofed through HTTP requests in
 </ol>
 </div>
 
+<h3 id=https-upgrades>HTTPS upgrading</h3>
```

Comment:
Needs more newlines before the `<h3>` I think. (Might in general want to double check this PR is not eating newlines in front of headings.)

Reply:
Done

Reply:
The end of this inserted section still eats two newlines from the subsequent `<h2>`. Please do review the entire PR for this.

Reply:
Done this time. Also fixed a mismatched <h3> tag.